README.md

```diff
@@ -100,11 +100,22 @@ class MyComponent extends Component {
 }
 ```
 
-Options:
+In order to prevent [unknown
+properties](https://facebook.github.io/react/warnings/unknown-prop.html) (and non-standard DOM
+attributes), you can use the `propsNamespace` option:
 
 ```js
-{
-  propsNamespace: 'translator', // injected props will be set in props.translator
+@withTranslator({ propsNamespace: 'translator' })
+class Presenter extends React.Component {
+  static propTypes = {
+    translator: PropTypes.shape({ __: PropTypes.func.isRequired }),
```

Comment:
Yes, let's either keep `object` or export a `translatorShape` that will be maintained.